pygromos/gromos/gromosPP.py

```diff
@@ -307,34 +307,42 @@ def frameout(self, in_top_path:str, in_coord_path:str, periodic_boundary_conditi
         """
 
         options = ""
-        if(out_file_format != None):
+        if(out_file_format is not None):
             options += "@outformat " + str(out_file_format) + " "
 
```

Comment:
I think I found an olde issue with line 339. this could also be used in 348. :) 
if out_file_format is None, there should be an error.

To solve this I suggest an else here, that is giving a default out_file_format.

I would suggest:
```
else:
    out_file_format = in_coord_path.split(".")[-1]
```